mne/time_frequency/tfr.py

```diff
@@ -1317,7 +1317,7 @@ def plot_joint(self, timefreqs=None, picks=None, baseline=None,
         cmap : matplotlib colormap
             The colormap to use.
         dB : bool
-            If True, 20*log10 is applied to the data to get dB.
+            If True, 10*log10 is applied to the data to get dB.
```

Comment:
There are two other occurrences in the file, can you fix those as well?

Reply:
sure